spec.md

```diff
@@ -1172,7 +1172,7 @@ For success responses, the following fields are defined:
 
 | Response Field | Type | Description |
 | --- | --- | --- |
-| dashboard_url | string | The updated URL of a web-based management user interface for the Service Instance; we refer to this as a service dashboard. The URL MUST contain enough information for the dashboard to identify the resource being accessed (`0129d920a083838` in the example below). Note: a Service Broker that wishes to return `dashboard_url` for a Service Instance MUST return it with the initial response to the update request, even if the Service Instance is being updated asynchronously. If present, MUST be a non-empty string. If not present, the Platform MUST retain the previous value of the `dashboard_url`. |
+| dashboard_url | string | The updated URL of a web-based management user interface for the Service Instance; we refer to this as a service dashboard. The URL MUST contain enough information for the dashboard to identify the resource being accessed (`0129d920a083838` in the example below). Note: a Service Broker that wishes to return `dashboard_url` for a Service Instance MUST return it with the initial response to the update request, even if the Service Instance is being updated asynchronously. If present, MUST be a string or null. If not present, the Platform MUST retain the previous value of the `dashboard_url`. |
```

Comment:
the last 2 sentences are awkward. Let's remove the part about it being a string since that's unnecessary. 
Does CF treat "null" the same as not present? Meaning the last sentence applies? If so, we should make it clear that null==not present.

Reply:
Yes, CC treats 'null' the same as not present for this specific field. We'll update the text to clarify.